go/exorelay/exorelay_test.go

```diff
@@ -27,16 +29,51 @@ func newExocom() *exocomMock.ExoComMock {
 	return exocom
 }
 
+func WaitFor(condition func() bool, errorMessage string) error {
```

Comment:
There might be, I didn't look for anything as it was basically implemented in on exocom-mock.